### PR TITLE
Refresh / Upgrade tokens

### PR DESCRIPTION
**Pull request by maxsam4**
This will allow Issuers to regenerate a fresh security token in the same ticker name if they want to do an upgrade or want to reset the data.

### CODE REVIEW

**Comment by satyamakgec** [comment]
This will only work when we are not deploying the STRProxy again? @maxsam4

**Comment by maxsam4** [comment]
> This will only work when we are not deploying the STRProxy again? @maxsam4

We will be upgrading the STR implementation contract rather than deploying STRProxy again.

**Review by satyamakgec — APPROVED** [review]

File: contracts/SecurityTokenRegistry.sol [hunk 1]
```diff
@@ -525,7 +528,38 @@ contract SecurityTokenRegistry is EternalStorage, Proxy {
...
+        bool _divisible,
+        address _treasuryWallet
+    )
+        public whenNotPausedOrOwner returns (address)
+    {
+        require(bytes(_name).length > 0 && bytes(_ticker).length > 0, "Bad ticker");
+        require(_treasuryWallet != address(0), "0x0 not allowed");
+        string memory ticker = Util.upper(_ticker);
+        bytes32 statusKey = Encoder.getKey("registeredTickers_status", ticker);
+        require(getBoolValue(statusKey), "not deployed");
```
Comment (adamdossa) [inline contracts/SecurityTokenRegistry.sol]: Could use `_tickerStatus` instead

**Review by adamdossa — APPROVED** [review]

File: contracts/SecurityTokenRegistry.sol (hunk 1 above)
Comment (maxsam4) [reply contracts/SecurityTokenRegistry.sol]: done, thanks for pointing out.

**Review by VictorVicente — APPROVED** [review]

**Pull request merged**